test/marshaling_test.ts

```diff
@@ -192,4 +197,14 @@ describe('Marshaling', () => {
     assert.throws(() => unmarshalValue(1), 'Not a marshaled value: 1');
     assert.throws(() => unmarshalValue({foo: 'bar'}), 'Not a marshaled value: {"foo":"bar"}');
   });
+
+  it('marshals values for logging', () => {
+    assert.deepEqual(transformForLogging(['one string']), ['one string']);
+    assert.deepEqual(transformForLogging(['two', 'strings']), ['two strings']);
+    assert.deepEqual(transformForLogging(['a %s c', 'b']), ['a b c']);
+    assert.deepEqual(transformForLogging(['%o vs %j', {a: 'b'}, {a: 'b'}]), [`{ a: 'b' } vs {"a":"b"}`]);
```

Comment:
Just an FYI, the "%o" here won't actually work for user packs because it seems like it's not supported by the pure-js version of `util.format` that the esbuild thunk uses. The `%j` does work, though

Reply:
Added comments in the code about this and a thunk_test to demonstrate it

Reply:
good to know. thanks for calling this out.